src/components/SwapTokenInput.tsx

```diff
@@ -57,6 +58,19 @@ export default function SwapTokenInput({
   const [popOverWidth, setPopOverWidth] = useState<number | undefined>()
   const theme = useTheme()
 
+  const filterOptions = (
+    options: TokenOption[],
+    { inputValue }: { inputValue: string },
+  ) =>
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-return, @typescript-eslint/no-unsafe-call
```

Comment:
not good enough at TS to figure this out

Reply:
I think it's because you didn't declare a return value. Want me to take a stab at it?